ofxClickDown_Example/src/ofApp.cpp

```diff
@@ -7,6 +7,17 @@ void ofApp::setup(){
     ofSetFrameRate(60);
     ofSetCircleResolution(40);
     ofEnableBlendMode(OF_BLENDMODE_ALPHA);
+<<<<<<< HEAD
```

Comment:
These arrow things followed by the `=======` (see line 20) and then another set of arrows (see line 30) mean that there was some conflict with a commit you were trying to merge with, and git is showing you the two different states using these markers to separate them. When git does this, it would have told you something like "merged but you need to resolve some conflicts", then it would have told you all the files and line numbers where there were conflicts so that you can go in and edit those files to however you want them (in most cases here it looks like you want to just delete the old version of the code and keep your new one). Normally you do this edit (aka resolve the conflicts) before committing, however it looks like you just committed with all the conflicts still here haha.

To fix this, you should be able to just go into these files, delete the old code (along with all the `<<<<<<< HEAD`, `=======` and `>>>>>>> asdfasdflkj` markers), commit those changes and then push to `origin master` again which will automatically update this PR.